src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/collectionsgrid/viewmodel/CollectionsGridViewModel.kt

```diff
@@ -28,10 +31,16 @@ class CollectionsGridViewModel: ViewModel() {
     // List of collection children (i.e. the chapters) to display in the list
     var children: ObservableList<Collection> = FXCollections.observableList(mutableListOf())
 
+    var tabs: ObservableList<Collection> = FXCollections.observableList(mutableListOf())
+
     // Selected child
     private var activeCollection: Collection by property()
     val activeCollectionProperty = getProperty(CollectionsGridViewModel::activeCollection)
 
+    private var selectedResourceClass: CssRule by property(MainScreenStyles.scripture)
```

Comment:
This `MainScreenStyles.scripture` could be put in the CollectionsGridStyles so that it's `CollectionGridStyles.sripture.` I think that makes more sense

Reply:
I didn't do this since I imagine we will use these styles for the takes page as well which will also have scripture/tN/etc styled tabs- what do you think?

Reply:
the only style that we should be accessing/using across different views/fragments should be AppStyles imo. I don't think we should have MainStyles  anywhere other than MainScreenView, CollectionGridStyles for CollectionGridView etc. AppStyles is the only exception

Reply:
@natibekele Do you think it makes sense to put the `scripture` and `translationNotes` rules in `AppStyles`? That may mean we'll have to import these rules in other stylesheets, but I think that would be better than defining the same css rules in multiple stylesheets